service/test/integration1_test.go

```diff
@@ -191,6 +191,9 @@ func Test1ClientServer_exit(t *testing.T) {
 }
 
 func Test1ClientServer_step(t *testing.T) {
+	if runtime.GOARCH == "arm64" {
```

Comment:
I find it hard to believe that this test doesn't work but TestClientServer_step without 1 does since they are the exact same tests but using different API.

Reply:
The remote debug code is not checked, and i tested it may failed contingently before(it may ok after https://github.com/go-delve/delve/pull/1733/commits/b1369a96231ebe9028e576fe61e60b593b655aa2). I think it should be skiped for arm64-delve check until remote debug released on arm64.